libs/prelude/Prelude.idr

```diff
@@ -37,6 +37,8 @@ import public Language.Reflection.Errors
 %default total
 
 -- Things that can't be elsewhere for import cycle reasons
+%runElab (defineFunction $ DefineFun `{void} [])
```

Comment:
Might it help to have a comment in `Builtins.idr` that the definition is in `Prelude.idr`?